test/fastly/client_test.rb

```diff
@@ -20,20 +20,17 @@
       assert_equal nil, client.password
     end
 
+    it 'raises an Unauthorized if api_key is not passed in the options' do
```

Comment:
I'm not sure if this was intended or not, but seemed to A.) read more clearly without 'an', while B.) being consistent with the format used in the test that followed this one...

```suggestion
    it 'raises Unauthorized if api_key is not passed in the options' do
```